From af5cc138782430963d05b2f92e60916f48c9f807 Mon Sep 17 00:00:00 2001
From: Kenneth Myhra <kennethmyhra@gmail.com>
Date: Mon, 1 May 2023 17:33:37 +0200
Subject: [PATCH 1/2] [compat.h] Recognize endianness of SerenityOS by picking
 up endian.h

We have endian.h so let's pick that header.
---
 include/compat.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/jfbuild/include/compat.h b/jfbuild/include/compat.h
index 8a879b0..66ddcad 100644
--- a/jfbuild/include/compat.h
+++ b/jfbuild/include/compat.h
@@ -100,7 +100,7 @@ typedef unsigned __int64 uint64_t;
 # define NULL ((void *)0)
 #endif
 
-#if defined(__linux)
+#if defined(__linux) || defined(__serenity__)
 # include <endian.h>
 # if __BYTE_ORDER == __LITTLE_ENDIAN
 #  define B_LITTLE_ENDIAN 1
-- 
2.40.1

